core/src/epicli/data/common/ansible/playbooks/roles/upgrade/tasks/node-exporter.yml

```diff
@@ -11,32 +7,60 @@
   include_vars:
     file: roles/node_exporter/vars/main.yml
 
-- name: Node Exporter | Test if node exporter service is configured
+- name: Node Exporter as DaemonSet | Get information about helm releases
+  delegate_to: localhost
+  become: false
+  run_once: true
+  environment: { KUBECONFIG: "{{ vault_location }}/../kubeconfig" }
+  command: helm list
+  register: check_node_exporter_release
+
+- name: Reinstall Node Exporter as DaemonSet from default namespace to custom namespace for "k8s as cloud service"
+  when:
+    - specification.helm_chart_name in check_node_exporter_release.stdout
```

Comment:
Isn't `specification.helm_chart_name in check_node_exporter_release.stdout` too wide of a condition? :thinking: What if user deploys a chart with a name that completely contains `specification.helm_chart_name`? :cat2:

Reply:
Changes added, now it looks good in my opinion. Please check.